### DISCUSSION TITLE
How to disable Player completely?

### DISCUSSION THREAD

**Discussion by egorksv** [katspaugh/wavesurfer.js#3709] [Q&A]
Hi All,

I only need Renderer part of Wavesurfer, is there any way to disable Player bit?

My use case - we are using automation to create cue sheets on the live stream (using backend), and need to display waveform/  peak data and regions / markers with Wavesurfer. Actual audio is played by HLS video player. Any suggestion re where to look at?

Many thanks in advance.

**Comment by katspaugh** [comment]
You can just pass an empty string in the URL.

Like this example but with an empty URL: https://wavesurfer.xyz/examples/?predecoded.js

```js
const wavesurfer = WaveSurfer.create({
  url: '',
  duration: 25.1,
  peaks: [
    0, 0.0023595101665705442, 0.012107174843549728, 0.005919494666159153, ...
  ],
  ...
}
```
